package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "ts-dot-prop",
-  "version": "2.1.0",
+  "version": "2.1.1",
```

Versions are bump automatically during publish step.

```suggestion
  "version": "2.1.0",
```